src/stretchy.js

```diff
@@ -193,61 +192,52 @@ const svgSpan = function(group, options) {
         attributes.push(["preserveAspectRatio", "none"]);
 
         svgNode = new domTree.svgNode([path], attributes);
+        span = buildCommon.makeSpan([], [svgNode], options);
 
     } else {
-        let width;
+        let widthClass;
         let align;
+        const spans = [];
 
         const [paths, gWidth, vbHeight, alignOne] = katexImagesData[label];
         const numSvgChildren = paths.length;
-        const innerSVGs = [];
         height = vbHeight / 1000;
         minWidth = gWidth;
 
         for (let i = 0; i < numSvgChildren; i++) {
             path = new domTree.pathNode(paths[i]);
-            attributes = [];
+
+            attributes = [["width", "400em"], ["height", height + "em"]];
+            attributes.push(["viewBox", `0 0 ${viewBoxWidth} ${vbHeight}`]);
```

Comment:
It'd be nice to make these variable names more consistent, maybe `vbWidth` and `vbHeight`.

Reply:
I can make the names consistent as `viewBoxWidth` and `viewBoxHeight`. To do that, I'll have to remove the `const` from the destructuring statement:
````
const [paths, gWidth, vbHeight, alignOne] = katexImagesData[label];
````